cognee/infrastructure/databases/vector/embeddings/LiteLLMEmbeddingEngine.py

```
@@ -48,22 +63,29 @@ async def exponential_backoff(attempt):
+                    "data": [{"embedding": [0.0] * self.dimensions} for _ in text]
+                }
                 return [data["embedding"] for data in response["data"]]
             else:
+                # Configure model name based on provider
+                if self.provider == "gemini":
+                    model_name = f"gemini/{self.model}"
+                    # For Gemini, we need to ensure we're using their specific endpoint format
+                    api_base = f"{self.endpoint}/models/{self.model}:embedContent"
+                else:
+                    model_name = self.model
+                    api_base = self.endpoint
```

Ruff (0.8.2)

Local variable `api_base` is assigned to but never used

Remove assignment to unused variable `api_base`

(F841)